app/services/moment.js

```diff
@@ -0,0 +1,34 @@
+import Ember from 'ember';
+import moment from 'moment';
+
+const { computed, defineProperty } = Ember;
+
+export default Ember.Service.extend({
+  init() {
+    this._super(...arguments);
+
+    let locale = null;
+
+    defineProperty(this, 'locale', computed({
```

Comment:
I would recommend `_locale`  on the instance instead.

DefProp comes at a non trivial cost, although for this service it will not be noticeable, I would prefer future travelers to not cargo cult this in other scenarios.